src/ica_api.c

```diff
@@ -2272,6 +2273,9 @@ unsigned int ica_des_ctr(const unsigned char *in_data, unsigned char *out_data,
 #ifdef ICA_FIPS
 	if (fips)
 		return EACCES;
+	if (ctr_width < (DES_BLOCK_SIZE*8) &&
```

Comment:
Blanks between DES_BLOCK_SIZE*8 -> DES_BLOCK_SIZE * 8
Not sure if the brackets are really needed.
Same at the other 2 changed places

Reply:
OK, will add the blanks. I copied the code piece from a few lines below where they coded it like this. Maybe we can do some general cleanup later with a separate commit. I would leave the brackets for better readability.